widgetastic_manageiq/__init__.py

```diff
@@ -1022,6 +1022,48 @@ def workaround_save_issue(self):
         )
 
 
+class ReactCodemirror(Widget):
+    """Looks the same as ScriptBox but it is changed in 5.11
+    https://github.com/JedWatson/react-codemirror"""
+
+    ROOT = "//div[contains(@class,'miq-codemirror')]//textarea"
+    PARENT_DIV = "./.."  # we need to change visibility of this one
+    CLICK_DIV = "./../.."  # the widget has very nested divs,we need to click on that one ¯\_(ツ)_/¯
+
+    def __init__(self, parent, logger=None):
+        Widget.__init__(self, parent, logger=logger)
+
+    def __locator__(self):
```

Comment:
@lina-nikiforova No need to define `__locator__` if you have defined a class constant of `ROOT` it will auto-resolve with the `Widget` class.

https://github.com/RedHatQE/widgetastic.core/blob/2bca6bfa4dfc5a098218ec0d9b251e1b3600bc92/src/widgetastic/widget/base.py#L648-L651

Reply:
@john-dupuy awesome, thanks! I didn't find it :)